storage/innobase/xtrabackup/doc/source/release-notes/2.4/2.4.12.rst

```diff
@@ -0,0 +1,39 @@
+===========================
+|Percona XtraBackup| 2.4.12
+===========================
+
+Percona is glad to announce the release of |Percona XtraBackup| 2.4.12 on
+June 21, 2018. Downloads are available from our `download site
+<http://www.percona.com/downloads/XtraBackup/Percona-XtraBackup-2.4.12/>`_ and
+from :ref:`apt <apt_repo>` and :ref:`yum <yum_repo>` repositories.
+
+All Percona software is open-source and free.
+
+New features and improvements
+=============================
+
+* |Percona XtraBackup| now prints used arguments to standard output. Bug
+  fixed :jirabug:`PXB-1494`.
+
+
+Bugs fixed
+==========
+
+* ``xtrabackup --copy-back`` didn't read plugins folder location from
```

Comment:
Lets say "didn't read which encryption plugin to use from `plugin-load` setting"... `plugin-load` doesn't really specify location (`plugin-dir` does), it only specifies plugin file names.